packages/server/src/helpers/iso/isoCrypto/getWebCrypto.ts

```diff
@@ -49,7 +49,7 @@ export const _getWebCryptoInternals = {
   stubThisImportNodeCrypto: async () => {
     try {
       // dnt-shim-ignore
-      const _nodeCrypto = await import('node:crypto');
+      const _nodeCrypto = await import(/* webpackIgnore: true */ 'node:crypto');
```

Comment:
I planned on adding a comment like this before you beat me to the PR for #517:

```suggestion
      /**
       * The `webpackIgnore` here is to help support Next.js' Edge runtime.
       * See https://github.com/MasterKale/SimpleWebAuthn/issues/517 for more info.
       */
      const _nodeCrypto = await import(/* webpackIgnore: true */ 'node:crypto');
```

Once this is in then this is good to go.